File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/DownloadLibrary.java
package de.dimedis.mobileentry.backend.response;

import java.io.File;

import de.dimedis.mobileentry.util.PrefUtils;

public class DownloadLibrary {
    public File libFile;

    public DownloadLibrary(File newFile) {
        this.libFile = newFile;
        PrefUtils.setLibraryVersion(newFile);
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/DownloadLibrary.kt
package de.dimedis.mobileentry.backend.response

import de.dimedis.mobileentry.util.PrefUtils
import java.io.File

class DownloadLibrary(var libFile: File?) {
    init {
        libFile?.let { PrefUtils.setLibraryVersion(it) }
    }
}